sdks/python/apache_beam/tests/pipeline_verifiers.py

```diff
@@ -72,11 +75,29 @@ def retry_on_io_error_and_server_error(exception):
     return False
 
 
+def retry_on_http_and_value_error(exception):
+  """Filter allowing retries on Bigquery errors and value error."""
+  if isinstance(exception, GoogleCloudError) or \
+          isinstance(exception, ValueError):
+    return True
+  else:
+    return False
+
+
+def compute_hash(list, hashing_alg=DEFAULT_HASHING_ALG):
+  """Compute a hash value from a list of string using SHA-1 as default."""
```

Comment:
There is no need to mention what the default is in this comment. Otherwise, if the default changes this comment will be stale.